app/views/accounts/_form.html.erb

```diff
@@ -23,7 +23,7 @@
 
 <div class="form-group">
   <%= f.label :role, "Rolle" %>
-  <%= f.select :role, User.roles.keys.map { |role| [role.humanize, role] }, {}, class: "form-control" %>
+  <%= f.select :role, User.roles.keys.map { |role| [role.camelcase, role] }, {}, class: "form-control" %>
```

Comment:
this change would be nice being documented in the commit. why was this necessary to change?

Reply:
Screenshot #1 
Naming of "ExtendedUser" in Dropdown would be "Extended user" with `.humanize`

Reply:
ok :/